### ISSUE TITLE
[Bug] Broken internet connection when using on windows hiddify in VPN mode

### ISSUE THREAD

**Issue by sat0sh1c** [glzr-io/zebar#159]
### Describe the bug

When hiddify is launched in a VPN mode the entire connection is extremely unstable and tend to stop working after some period of time. I am sure that this issue is related to this app, as If I close it the connection is working completely fine

### Reproduction

Download [hiddify](https://github.com/hiddify/hiddify-app/releases) in settings select VPN mode to any server. 
P.S: The connection was made via vless protocol

### Version number

2.5.1

**Comment by lars-berger** [comment]
Which providers do you use in your config? If you're running one of the default configs, could you see if the issue's still reproducible with the `network` provider commented out?

**Comment by Satoshics** [comment]
> Which providers do you use in your config? If you're running one of the default configs, could you see if the issue's still reproducible with the `network` provider commented out?

Yeah, that seems to solve it, I guess the network is the reason for break. Moreover when the network was enable the Wi-Fi icon was crossed. like there was no connection.

**Comment by fevzibabaoglu** [comment]
> Moreover when the network was enable the Wi-Fi icon was crossed. like there was no connection.

I am also experiencing this when using Cloudflare WARP. There seems to be an issue with `NetworkInterface.interface_type`. It returns as `undefined` when the default interface is a tunnel type, instead of correctly identifying it as `tunnel`. While WARP is active, there are (at least) two interfaces active: WIFI and the WARP tunnel. Ideally, WIFI or Ethernet should be prioritized as the `defaultInterface`. However, the WARP tunnel is selected as the `defaultInterface`, and its `interface_type` is not properly defined.

**Comment by MarcosCosmos** [comment]
@fevzibabaoglu I believe the default interface is derived from windows, and outside of that provision I would say that selecting which interface to show on the bar is really a matter for the client side JS.

**Comment by MarcosCosmos** [comment]
#259 Provides more details, and appears to be the same issue

**Comment by MarcosCosmos** [comment]
Closing as duplicate of #259
